hostpath-provisioner_test.go

```diff
@@ -79,13 +77,33 @@ func Test_calculatePvCapacity(t *testing.T) {
 	type args struct {
 		path string
 	}
+	// Get total size of wherever we are running.
+	capacity, _ := getTotalCapacity(".")
```

Comment:
This works, but aren't you kind of relying on the function your testing to determine the results of the test?